docs/how-to-guides/tracing/trace_span.md

```diff
@@ -0,0 +1,150 @@
+
+This document outlines the design of PromptFlow spans, detailing what information is traced and how it is structured. By adhering to these specifications, we ensure transparency and consistency in our tracing system.
+
+The UI interprets the captured spans and presents them in a user-friendly manner. Understanding the fields and contracts defined within the spans is essential for effectively utilizing PromptFlow or integrating its components.
+
+# PromptFlow Span
+
+## OpenTelemetry Span Basics
+
+A typical span object contains below information:
+
+| Field | Description |
+|---|---|
+| name | Name of span |
+| parent_id | Parent span ID (empty for root spans) |
+| context | [Span Context](https://opentelemetry.io/docs/concepts/signals/traces/#span-context) |
+| start_time | Start time of the span |
+| end_time | End time of the span |
+| status | [Span Status](https://opentelemetry.io/docs/concepts/signals/traces/#span-status) |
+| attributes | [Attributes](https://opentelemetry.io/docs/concepts/signals/traces/#attributes) |
+| events | [Span Events](https://opentelemetry.io/docs/concepts/signals/traces/#span-events) |
+| links | [Span Links](https://opentelemetry.io/docs/concepts/signals/traces/#span-links) |
+
+## PromptFlow Span Specification
+
+In PromptFlow, we define several span types, and the system automatically creates spans with execution information in designated attributes and events.
+
+These span types share common attributes and events, which we refer to as standard attributes and events. Let’s explore these common elements before diving into the specifics of each span type.
+
+**Standard Attributes**
+
+Each span in PromptFlow is enriched with a set of standard attributes that provide essential information about the span's context and purpose. The following table outlines these attributes:
+
+| Attribute | Type | Description | Examples | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) |
+|---|---|---|---|---|
+| framework | string | Identifies the framework within which the trace is captured.  | promptflow | `Required` |
+| node_name | string | Denotes the name of the flow node. | chat | `Conditionally Required` if the flow is a Directed Acyclic Graph (DAG) flow. |
+| span_type | string | Specifies the type of span, such as LLM or Flow. | LLM | `Required` |
+| line_run_id | string | Unique identifier for the execution run within PromptFlow. | d23159d5-cae0-4de6-a175-295c715ce251 | `Required` |
+| function | string | The function associated with the span. | search | `Recommended` |
+| session_id | string | Unique identifier for chat sessions. | 4ea1a462-7617-439f-a40c-12a8b93f51fb | `Opt-In` |
+| referenced.line_run_id | string | Represents the line run ID that is the source of the evaluation run. | f747f7b8-983c-4bf2-95db-0ec3e33d4fd1 | `Conditionally Required`: Only used in evaluation runs |
```

Comment:
Add a link to the document which explains what is an evaluation run?

Reply:
Added